### PR TITLE
Add deprecation notices to: library, readme, docs, CLI

### PR DESCRIPTION
**Pull request by mikeb-stripe**
## Proposed changes
Add deprecation notices to: library (console.warn), Readme, docs site, and CLI

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# yarn
+yarn.lock
```
Comment (charliecruzan-stripe) [inline .gitignore]: I think usually the `yarn.lock` file is committed

File: README.md [hunk 2]
```diff
@@ -1,9 +1,8 @@
-# tipsi-stripe
-
-This library isn't actively maintained now. Please, consider usage of Stripe's official library:
+# 🚨 This library is no longer maintained 🚨
 
-https://github.com/stripe/stripe-react-native
+If you're building mobile apps with react-native, please use Stripe's official library, [`stripe-react-native`](https://github.com/stripe/stripe-react-native).
```
Comment (charliecruzan-stripe) [inline README.md]:
```suggestion
If you're building apps with React Native, please use [Stripe's official React Native library](https://github.com/stripe/stripe-react-native).
```

File: postinstall.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+/* eslint-disable no-console */
+console.warn(
+  '🚨 [tipsi-stripe] Deprecation notice: tipsi-stripe is no longer maintained. Please migrate your project to stripe-react-native: https://github.com/stripe/stripe-react-native'
```
Comment (charliecruzan-stripe) [inline postinstall.js]:
```suggestion
  '🚨 [tipsi-stripe] Deprecation notice: tipsi-stripe is no longer maintained. Please migrate your project to @stripe/stripe-react-native: https://github.com/stripe/stripe-react-native'
```

File: src/index.js [hunk 4]
```diff
@@ -5,3 +5,8 @@ import errorCodes from './errorCodes'
 export { PaymentCardTextField, errorCodes }
 
 export default Stripe
+
+// eslint-disable-next-line no-console
+console.warn(
+  '🚨 [tipsi-stripe] Deprecation notice: tipsi-stripe is no longer maintained. Please migrate your project to stripe-react-native: https://github.com/stripe/stripe-react-native'
```
Comment (charliecruzan-stripe) [inline src/index.js]:
```suggestion
  '🚨 [tipsi-stripe] Deprecation notice: tipsi-stripe is no longer maintained. Please migrate your project to @stripe/stripe-react-native: https://github.com/stripe/stripe-react-native'
```

File: website/docs-md/index.md [hunk 5]
```diff
@@ -1,27 +1,9 @@
...
-* SDK 17+
-
-## Testing Payments
-
-**Credit Cards**: you can use real cards for payments on test environment without hesitation. You won't be charged on test environment for card payments.
-
-Or you can use [test cards provided by stripe](https://stripe.com/docs/testing#cards).
-
-**Google Pay**: will charge you for $1 but return money soon (~1hr).
+## 🚨 Deprecation notice: please use stripe-react-native 🚨
```
Comment (charliecruzan-stripe) [inline website/docs-md/index.md]:
```suggestion
## 🚨 Deprecation notice: please use `@stripe/stripe-react-native` 🚨
```

File: website/docs-md/index.md [hunk 6]
```diff
@@ -1,27 +1,9 @@
...
 
+This library is no longer maintained. Please update your project to use [`stripe-react-native`](https://github.com/stripe/stripe-react-native)
```
Comment (charliecruzan-stripe) [inline website/docs-md/index.md]:
```suggestion
This library is no longer maintained. Please update your project to use [`@stripe/stripe-react-native`](https://github.com/stripe/stripe-react-native)
```

File: website/docs-md/start.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
+---
+id: start
+title: (Deprecated) Start here
+sidebar_label: (Deprecated) Start here
+---
+
+## 🚨 Deprecation notice: please use stripe-react-native 🚨
```
Comment (charliecruzan-stripe) [inline website/docs-md/start.md]:
```suggestion
## 🚨 Deprecation notice: please use `@stripe/stripe-react-native` 🚨
```

File: website/docs-md/start.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+This library is no longer maintained. Please update your project to use [`stripe-react-native`](https://github.com/stripe/stripe-react-native)
```
Comment (charliecruzan-stripe) [inline website/docs-md/start.md]:
```suggestion
This library is no longer maintained. Please update your project to use [`@stripe/stripe-react-native`](https://github.com/stripe/stripe-react-native)
```

File: website/docs/pages/en/index.js [hunk 9]
```diff
@@ -79,32 +79,32 @@ class HomeSplash extends React.Component {
         <div className="inner">
           <ProjectTitle />
           <PromoSection>
-            <Button href="https://github.com/tipsi/tipsi-stripe/tree/master/example">
-              Sample Project
+            <Button href="https://github.com/stripe/stripe-react-native">
+              Official library: stripe-react-native
```
Comment (charliecruzan-stripe) [inline website/docs/pages/en/index.js]:
```suggestion
              Official library: @stripe/stripe-react-native
```

File: website/docs/pages/en/index.js [hunk 10]
```diff
@@ -113,29 +113,36 @@ const Showcase = (props) => {
...
+    ));
 
   return (
-      <div className="productShowcaseSection paddingBottom">
+    <div className="productShowcaseSection paddingBottom">
       <h2>Quick navigation</h2>
-      <a href="/tipsi-stripe/docs/index.html">Start reading docs</a>
+      <p>
+        <a href="https://github.com/stripe/stripe-react-native">
+          Official library: stripe-react-native
```
Comment (charliecruzan-stripe) [inline website/docs/pages/en/index.js]:
```suggestion
          Official library: @stripe/stripe-react-native
```

**Review by charliecruzan-stripe — APPROVED** [review]
looks good! thanks 🙏 

only suggested changes are going from `stripe-react-native` to `@stripe/stripe-react-native` which is the actual npm package name (but i would be fine with either/or being used here)

File: .gitignore (hunk 1 above)
Comment (mikeb-stripe) [reply .gitignore]: Yep, fair point - this project uses `npm` apparently, and I use `yarn` out of habit. Didn't mean to check this in. Good catch!

**Pull request merged**